framework/materials/mat_prop_scalarfuncXYZTV.cc

```diff
@@ -6,7 +6,7 @@
 namespace opensn
 {
 
-OpenSnRegisterObject(chi_objects, MaterialPropertyScalarFuncXYZTV);
+OpenSnRegisterObjectNamespace(objects, MaterialPropertyScalarFuncXYZTV);
```

Comment:
I think this helps clarifying what is being registered.

Reply:
I'm not sure I agree with this. This sounds like it is registering a namespace for an object, not registering an object in a namespace. This creates `objects.MaterialPropertyScalarFuncXYZTV` and the associated `Create` routine. Are we calling  `objects.MaterialPropertyScalarFuncXYZTV` a namespace?